core/src/main/resources/defaults/blueprints/hdf31-flow-management.bp

```diff
@@ -0,0 +1,92 @@
+{
+  "inputs": [],
+  "description": "Useful for data-flow management with Apache NiFi",
+  "blueprint": {
+    "Blueprints": {
+      "blueprint_name": "hdf31-flow-management",
+      "stack_name": "HDF",
+      "stack_version": "3.1"
+    },
+    "configurations": [
+      {
+        "nifi-ambari-config": {
+          "nifi.security.encrypt.configuration.password": "changemeplease",
```

Comment:
I'm not sure if it is a good idea to have password overrides in this file.

Reply:
Those passwords are required to be set when installing with Ambari.  Most of this work was done in CloudBreak 1.16, and at that time I was not aware of a way to have CloudBreak prompt the user for that type of information.  Is there a way to allow the user to provide this information without putting it in the blueprint?